cmd/client_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Google LLC // // Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
```

Comment:
Nit: remove the Author entry. 1) There will be more than one author most probably. 2) Authorship can be traced back via commit logs.

Reply:
I have removed the author entry from every file.